src/site/paragraphs/linkTeaser.drupal.liquid

```diff
@@ -33,16 +33,20 @@
   data-links-list-header="{{ link.title | escape }}"
   data-links-list-section-header="{{ section_header | escape }}"
   {% if parentFieldName === 'field_spokes' %}class="hub-page-link-list__item"{% endif %}>
-    <a href="{{link.url.path}}"
-      {% if linkTeaser.options["target"] %}target="{{ linkTeaser.options["target"] }}"{% endif %}
-      >
-        {% if link.title != empty %}
-            <{{ header }} class="{{ headerClass }}">{{ link.title }}</{{ header }}>
-            {% if parentFieldName === "field_spokes" %}
-                <img class="all-link-arrow" src="/img/arrow-right-blue.svg" alt="right-arrow">
-            {% endif %}
-        {% endif %}
-    </a>
+    {% if link.title != empty %}
+      <{{ header }} class="{{ headerClass }}">
+        <a href="{{link.url.path}}" class="vads-u-text-decoration--underline"
+          {% if linkTeaser.options["target"] %}
+            target="{{ linkTeaser.options["target"] }}"
+          {% endif %}
+        >
+          {{ link.title }}
+        </a>
+      </{{ header }}>
+      {% if parentFieldName === "field_spokes" %}
+          <img class="all-link-arrow" src="/img/arrow-right-blue.svg" alt="right-arrow">
```

Comment:
Could we update the `alt` text here to be more descriptive. Maybe "Blue arrow pointing right" (I haven't seen the image)

Reply:
+1 to Tim's comment. Here's a decent article on `alt` text. 

https://webaim.org/techniques/alttext/